osdep/io.c

```diff
@@ -255,31 +372,162 @@ int mp_printf(const char *format, ...)
 
 int mp_open(const char *filename, int oflag, ...)
 {
-    int mode = 0;
-    if (oflag & _O_CREAT) {
-        va_list va;
-        va_start(va, oflag);
-        mode = va_arg(va, int);
-        va_end(va);
```

Comment:
So the mode argument just gets ignored?

Reply:
Yeah, POSIX modes don't really map to Windows' security model. MSVCRT only uses S_IWUSR to set FILE_ATTRIBUTE_READONLY and ignores other mode bits. (We could add this feature.)